s3/compress.go

```diff
@@ -134,7 +191,19 @@ func (c *Compressor) Compress(ctx context.Context, m *models.ExportPayload) (tim
 	filename := fmt.Sprintf("%s-%s.tar.gz", t.Format(time.RFC3339), m.ID.String())
 	s3key := fmt.Sprintf("%s/%s", m.OrganizationID, filename)
 
-	err := c.zipExport(ctx, t, prefix, filename, s3key)
+	sources, err := m.GetSources()
+	if err != nil {
+		return t, filename, s3key, fmt.Errorf("failed to get sources: %w", err)
+	}
+
+	meta := ExportMeta{
+		ExportBy:    m.User.Username,
+		ExportDate:  m.CreatedAt.Format(time.RFC3339),
+		ExportOrgID: m.User.OrganizationID,
+		HelpString:  helpString,
+	}
```

Comment:
Should building this object be the responsiblity of the zipExport() method?  Just something to think about.

Reply:
I went back and forth on that one. I ended up doing it this way, so it wouldnt pass down the ExportPayload model.

Reply:
I think its fine as it is.